iib/web/migrations/versions/04dd7532d9c5_polymorphic_requests.py

```diff
@@ -0,0 +1,382 @@
+    sa.Column('from_index_resolved_id', sa.Integer()),
+    sa.Column('index_image_id', sa.Integer()),
+)
+
+
+request_bundle_table = sa.Table(
+    'request_bundle',
+    sa.MetaData(),
+    sa.Column('request_id', sa.Integer()),
+    sa.Column('image_id', sa.Integer()),
+)
+
+
+request_add_bundle_table = sa.Table(
+    'request_add_bundle',
+    sa.MetaData(),
+    sa.Column('request_add_id', sa.Integer()),
+    sa.Column('image_id', sa.Integer()),
+)
+
+
+request_operator_table = sa.Table(
+    'request_operator',
+    sa.MetaData(),
+    sa.Column('request_id', sa.Integer()),
+    sa.Column('operator_id', sa.Integer()),
+)
+
+
+request_rm_operator_table = sa.Table(
+    'request_rm_operator',
+    sa.MetaData(),
+    sa.Column('request_rm_id', sa.Integer()),
+    sa.Column('operator_id', sa.Integer()),
+)
+
+
+def upgrade():
+    op.create_table(
+        'request_add',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('organization', sa.String(), nullable=True),
+        sa.Column('binary_image_id', sa.Integer(), nullable=False),
+        sa.Column('binary_image_resolved_id', sa.Integer(), nullable=True),
+        sa.Column('from_index_id', sa.Integer(), nullable=True),
+        sa.Column('from_index_resolved_id', sa.Integer(), nullable=True),
+        sa.Column('index_image_id', sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(['binary_image_id'], ['image.id']),
+        sa.ForeignKeyConstraint(['binary_image_resolved_id'], ['image.id']),
+        sa.ForeignKeyConstraint(['from_index_id'], ['image.id']),
+        sa.ForeignKeyConstraint(['from_index_resolved_id'], ['image.id']),
+        sa.ForeignKeyConstraint(['id'], ['request.id']),
+        sa.ForeignKeyConstraint(['index_image_id'], ['image.id']),
+        sa.PrimaryKeyConstraint('id'),
+    )
+    op.create_table(
+        'request_add_bundle',
+        sa.Column('request_add_id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.Column('image_id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.ForeignKeyConstraint(['image_id'], ['image.id']),
+        sa.ForeignKeyConstraint(['request_add_id'], ['request_add.id']),
+        sa.PrimaryKeyConstraint('request_add_id', 'image_id'),
+        sa.UniqueConstraint('request_add_id', 'image_id'),
+    )
+    with op.batch_alter_table('request_add_bundle') as batch_op:
+        batch_op.create_index(
+            batch_op.f('ix_request_add_bundle_image_id'), ['image_id'], unique=False,
+        )
+        batch_op.create_index(
+            batch_op.f('ix_request_add_bundle_request_add_id'), ['request_add_id'], unique=False,
+        )
+
+    op.create_table(
+        'request_rm',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('binary_image_id', sa.Integer(), nullable=False),
+        sa.Column('binary_image_resolved_id', sa.Integer(), nullable=True),
+        sa.Column('from_index_id', sa.Integer(), nullable=True),
```

Comment:
As mentioned in a comment in `models.py`, could you please set `nullable=False` here?